ads/adblade.md

```diff
@@ -21,8 +21,8 @@ limitations under the License.
 ```html
 <amp-ad width=300 height=250
     type="adblade"
-    width="300"
-    height="250"
+    data-width="300"
+    data-height="250"
```

Comment:
please also update the supported param part from `height` to `data-height`, same with width. 
Also I would suggest using `data-height` instead of data-height.